targetFunctions/common/checkIndices.m

```diff
@@ -0,0 +1,60 @@
+function checkIndices(problemStruct)
```

Comment:
This is a duplicate of _API/checkIndicies.m_

Reply:
Yeah, I moved it and forgot to remove it from the repo.